src/components/Balances.jsx

```diff
@@ -68,8 +68,11 @@ class Balances extends Component {
                           <td className="td-donations-amount">
                             {roundBigNumber(
                               dc.currentBalance / (this.state.conversionRates[dc.symbol] || 1),
-                              (tokenWhitelist.find(t => t.symbol === this.state.currency) || {})
-                                .decimals || 2,
+                              (
+                                nativeCurrencyWhitelist.find(
```

Comment:
Let's change userProvider to solve this once! it's not cool to every time iterate over this list to get the currency precision

Reply:
Done